documentation/buses.yml

```diff
@@ -0,0 +1,272 @@
+  /buses/{busId}:
+    put:
+      parameters:
+        - name: "Accept-Language"
+          in: "header"
+          default: en
+          required: true
+          description: Preferred language
+        - name: busId
+          in: path
+          required: true
+          description: The bus ID
+      security:
+        - bearerAuth: []
+      tags:
+        - Buses
+      description: Update bus by Id
+      requestBody:
+        description: Request body
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/NewBus"
+      responses:
+        200:
+          description: Bus updated
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/ResBody"
+                  - type: object
+                    properties:
+                      data:
+                        type: object
+                        properties:
+                          count:
+                            type: number
+    delete:
+      parameters:
+        - name: "Accept-Language"
+          in: "header"
+          default: en
+          required: true
+          description: Preferred language
+        - name: busId
+          in: path
+          required: true
+          description: The bus ID
+      security:
+        - bearerAuth: []
+      tags:
+        - Buses
+      description: Delete bus by Id
+      responses:
+        200:
+          description: Bus deleted
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/ResBody"
+                  - type: object
+                    properties:
+                      data:
+                        type: object
+                        properties:
+                          count:
+                            type: number
+  /buses/assign:
+    post:
+      parameters:
+        - name: "Accept-Language"
+          in: "header"
+          default: en
+          required: true
+          description: Preferred language
+      security:
+        - bearerAuth: []
+      tags:
+        - Buses
+      description: Assign driver to bus
+      requestBody:
+        description: Request body
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/AssignBus"
+      responses:
+        201:
+          description: Driver assigned to bus
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/ResBody"
+                  - type: object
+                    properties:
+                      data:
+                        type: object
+                        properties:
+                          count:
+                            type: number
+  /buses/drivers:
```

Comment:
/buses/drivers? Try to make it meaningful. Try something like: /buses/buses-drivers